backend/tests/test_models.py

```diff
@@ -18,20 +23,132 @@ def test_delete_lesson_with_exercises(self, db_session, lesson_with_exercise):
         )
         assert not deleted_lesson
 
-    def test_progress_cascade_deleted_for_lesson(self):
+    def test_progress_cascade_deleted_for_lesson(self, db_session):
         """
         Progress records should be deleted if their lesson object was deleted.
         """
-        pass
 
-    def test_progress_cascade_deleted_for_user(self):
+        user = User(
```

Comment:
user is created with same attributes, so can be made as a fixture and used in other tests